core/src/sigverify.rs

```diff
@@ -44,14 +44,14 @@ impl SigVerifier for TransactionSigVerifier {
         &self,
         mut batches: Vec<PacketBatch>,
         valid_packets: usize,
-    ) -> Vec<PacketBatch> {
-        sigverify::ed25519_verify(
+    ) -> (Vec<PacketBatch>, usize) {
+        let count_invalid_packages = sigverify::ed25519_verify(
```

Comment:
Can we call it batches or packets? We don't use packages anywhere. Try to be consistent with the naming already present.